index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style2.css" />
+    <title>Chester</title>
+  </head>
+  <body>
+    <header>
+      <!-- mobile  -->
+      <div class="header-sect">
+        <span class="clas"></span>
+        <div id=mob-nav>
+          <ul class="menu-1">
+            
+             
+              <li><a href="#">Program</a></li>
+              <li> <a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li><li>
+           <a href="about.html">About</a></li>
+          </ul>
+        </div>
+        <span class="clas"></span>
+      </div>
+      <!-- Desktop  -->
+      <nav>
+        <div class="media">
+          <ul>
+            <li>
+              <a href="https://web.facebook.com/rotshidzwa.mavhungu.5/"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/footballlz"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a>English</a></li>
+            <li><a>My page</a></li>
+            <li><a>Logout</a></li>
+          </ul>
+        </div>
+        <div class="navba">
+          <a href="index.html"><img src="images/Capture.png" class="navImage" alt="" /></a>
+          <ul>
+            <li><a href="#">program</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="./about.html">about</a></li>
+            <li><button type="button">CT Campaign</button></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main> 
+      <section class="ban">
+        <h1>The 2022 
+            <br>sports
+             <br>celebration</h1>
```

Comment:
- [ ] I think we have a problem with the heading as seen in the image below. However, when I remove `background-clip: content-box;` style it looks fine.

![image](https://user-images.githubusercontent.com/4247740/164844976-d74ffd4b-7790-49e2-8445-bd5f8b1ecd10.png)